whatsapp_theming/README.md

```diff
@@ -0,0 +1,17 @@
+# whatsapp_theming
+An example app which demonstrates dark light and auto switching of theme options just like what whatsapp provides
+
+## Getting Started
+
+I have used shared preferences and theme provider package to develop such kind of dynamic theme changing hope it helps!
```

Comment:
Please don't write readme description like you are talking to someone. This line is not required at all.